planner:
  ros__parameters:
    use_hardware: false
    record_data: false
    num_batch: 500
    num_steps: 15
    maxiter_cem: 1
    maxiter_projection: 5
    w_pos: 3.0
    w_rot: 0.5
    w_col: 500.0
    num_elite: 0.05
    timestep: 0.1
    position_threshold: 0.06
    rotation_threshold: 0.1